packages/base/src/3dview/mainview.tsx

```diff
@@ -996,8 +996,12 @@ export class MainView extends React.Component<IProps, IStates> {
       }
 
       const boundingBox = selectedMesh.getObjectByName(SELECTION_BOUNDING_BOX);
```

Comment:
Last comment 😁 this will eventually never return anything now that you're not adding the box to the mesh, let's do only

```typescript
      const groupBoundingBox = this._meshGroup?.getObjectByName(
        SELECTION_BOUNDING_BOX
      );
```

Reply:
Thanks for making it better 🚀